content/nldi.md

```diff
@@ -2,7 +2,6 @@
 author: David Blodgett
 date: 2017-04-22
 slug: nldi-intro
-draft: True
```

Comment:
Can you put this back in so we can have one last look on dev before pushing it to prod?